Cargo.toml

```
@@ -4,10 +4,15 @@ version = "0.1.0"
 authors = ["Gear Technologies"]
 edition = "2021"
 description = "A wrapper over gstd that provides a fluent interface to interact with the Gear Protocol"
-repository = "https://github.com/StackOverflowExcept1on/gstd-fluent"
-license = "GPL-3.0"
+repository = "https://github.com/gear-foundation/gstd-fluent"
+license = "MIT"
 keywords = ["gstd", "smart-contracts", "gear-tech", "gear-foundation"]
 categories = ["api-bindings", "no-std"]
 
 [dependencies]
-gstd = { git = "https://github.com/gear-tech/gear.git", branch = "av/gstd-panic-handler", default-features = false }
+gstd = { git = "https://github.com/gear-tech/gear.git", rev = "f4407fa" }
```

Could we use `946ac47` rev as all other contracts are at it?
```suggestion
gstd = { git = "https://github.com/gear-tech/gear.git", rev = "946ac47" }
```